### PR TITLE
Add Importing Kickstart Repositories for Red Hat Enterprise Linux 7 s…

### PR DESCRIPTION
**Pull request by spetrosi**
…ection

Add a parent assembly Importing Kickstart Repositories and reuse the same module in it twice

Bug 1816488 - [DDF] I was testing this for a customer

https://bugzilla.redhat.com/show_bug.cgi?id=1816488

### CODE REVIEW

File: guides/common/modules/proc_importing-kickstart-repositories.adoc [hunk 1]
```diff
@@ -1,47 +1,62 @@
 [id="importing-kickstart-repositories_{context}"]
-= Importing Kickstart Repositories
+= Importing Kickstart Repositories for Red{nbsp}Hat Enterprise Linux{nbsp}{rhel-version}
 
-Kickstart repositories are not provided by the Content ISO image. To use Kickstart repositories in your disconnected {Project}, you must download a binary DVD ISO file for the version of Red{nbsp}Hat Enterprise Linux that you want to use and copy the Kickstart files to {Project}.
+To import kickstart repositories for Red{nbsp}Hat Enterprise Linux{nbsp}{rhel-version}, complete the following steps on {Project}.
```
Comment (suyogsainkar) [inline guides/common/modules/proc_importing-kickstart-repositories.adoc:4]: Capitalization not consistent with other instances. Kickstart?

File: guides/common/modules/proc_importing-kickstart-repositories.adoc [hunk 2]
```diff
@@ -1,47 +1,62 @@
...
-# mkdir /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/kickstart
 
-# mkdir /var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/baseos/kickstart
-----
-+
-Note that if you use Red Hat Enterprise Linux 7, you must create and complete all the following steps in only one directory `/var/www/html/pub/sat-import/content/dist/rhel/server/7/7.7/x86_64/kickstart/`.
-+
-. To the listing files `/var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/appstream/listing` and `/var/www/html/pub/sat-import/content/dist/rhel8/8.1/x86_64/baseos/listing`, append `kickstart` with a new line:
+ifeval::["{rhel-version}" == "8"]
+. Create directories for Red{nbsp}Hat Enterprise Linux 8 AppStream and BaseOS kickstart repositories:
```
Comment (suyogsainkar) [inline guides/common/modules/proc_importing-kickstart-repositories.adoc:37]: Same as above - Kickstart?

**Comment by suyogsainkar** [comment]
@spetrosi just one minor comment, rest all looks good to me.

File: guides/common/modules/proc_importing-kickstart-repositories.adoc (hunk 2 above)
Comment (spetrosi) [reply guides/common/modules/proc_importing-kickstart-repositories.adoc]: Done

File: guides/common/modules/proc_importing-kickstart-repositories.adoc (hunk 1 above)
Comment (spetrosi) [reply guides/common/modules/proc_importing-kickstart-repositories.adoc]: Done

**Comment by suyogsainkar** [comment]
@spetrosi looks good to me; merging.

**Review by suyogsainkar — APPROVED** [review]

**Pull request merged**